### ISSUE TITLE
Provide a better error message if gpg is missing

### ISSUE THREAD

**Issue by sweh** [flyingcircusio/batou#63]
On a new MacOS Cataline, batou fails decrypting secrets with the message

```
        File "/Users/fc/develop/claimx_deployment/.batou/lib/python2.7/site-packages/batou/secrets/encryption.py", line 110, in _decrypt
          shell=True)
        File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/subprocess.py", line 216, in check_output
          process = Popen(stdout=PIPE, *popenargs, **kwargs)
        File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/subprocess.py", line 394, in __init__
          errread, errwrite)
        File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/subprocess.py", line 1047, in _execute_child
          raise child_exception
      TypeError: execv() arg 2 must contain only strings
```

Can batou check if gpg is installed and provide a better error message?

**Comment by ctheune** [comment]
this should be easy to do using shutil.which() (new in 3.3) and then giving an appropriate message.

**Comment by ctheune** [comment]
Fixed in revision a1d75982931bb40156f6db3a908d912cdde256a6
